Project.toml

```diff
@@ -31,6 +24,10 @@ Tensors = "1"
 WriteVTK = "1.13"
 julia = "1.6"
 
+[extensions]
```

Comment:
I guess this is from using 1.8. A bit unfortunate that it reorganizes the sections. But anyway, should probably put it back to the way it was.

Reply:
So I guess the ordering of the sections matter in 1.9?

Reply:
The ordering shouldn't matter, just that 1.8 don't know how to sort this section so it puts it at the end.